core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -769,7 +772,12 @@ private[spark] object AppStatusStore {
   def createLiveStore(
       conf: SparkConf,
       appStatusSource: Option[AppStatusSource] = None): AppStatusStore = {
-    val store = new ElementTrackingStore(new InMemoryStore(), conf)
+    val storePath = conf.get(LIVE_UI_LOCAL_STORE_DIR).map(new File(_))
+    // For the disk-based KV store of live UI, let's simply make it ROCKSDB only for now,
+    // instead of supporting both LevelDB and RocksDB. RocksDB is built based on LevelDB with
```

Comment:
not related to this PR, I'm wondering why did we provide these two choices in the first place. RocksDB only seems sufficient.

Reply:
https://issues.apache.org/jira/browse/SPARK-18085?focusedCommentId=16088758&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-16088758

There were legal concerns.

Reply:
According to the context, Reynold mentioned that the legal issue was resolved. Do you mean that Apache Spark still have legal issue with RocksDB, @gengliangwang ?

![Screenshot 2022-11-23 at 12 28 24 PM](https://user-images.githubusercontent.com/9700541/203640432-10e7af59-ccb1-4bf3-811b-1b4b606410b4.png)

Reply:
@dongjoon-hyun I was explaining why @vanzin chose LevelDB. Maybe I was wrong about his reason.

> Do you mean that Apache Spark still have legal issue with RocksDB

I don't think so. If so, I wouldn't choose it as the only disk backend of live UI.

Reply:
Got it. Thank you for confirming that.